client/replays/replay-drop.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useEffect } from 'react'
+import { useAppDispatch } from '../redux-hooks'
+import { openReplay } from './action-creators'
+
+export default function ReplayDrop() {
+  const dispatch = useAppDispatch()
+
+  const onDrop = useCallback(
+    (e: DragEvent) => {
+      e.preventDefault()
+      e.stopPropagation()
+
+      const [file] = e.dataTransfer?.files || []
+      if (file?.path.endsWith('.rep')) {
```

Comment:
Same as during application launch, this needs to handle extensions case-insensitively